sockeye/decoder.py

```diff
@@ -1006,19 +891,11 @@ def __init__(self,
                               "We need to have the same number of hidden units in the decoder "
                               "as we have in the encoder")
 
-        if embed_weight is None:
-            embed_weight = encoder.Embedding.get_embed_weight(self.config.vocab_size,
-                                                              self.config.num_embed,
-                                                              C.TARGET_EMBEDDING_PREFIX)
-
-        self.embedding = encoder.Embedding(self.config.num_embed,
-                                           self.config.vocab_size,
-                                           prefix=C.TARGET_EMBEDDING_PREFIX,
-                                           embed_weight=embed_weight,
-                                           dropout=config.embed_dropout)
         self.pos_embedding = encoder.get_positional_embedding(config.positional_embedding_type,
-                                                              config.num_embed,
+                                                              num_embed=config.num_embed,
                                                               max_seq_len=config.max_seq_len_target,
+                                                              fixed_scale_input=False,
+                                                              fixed_scale_positions=False,
```

Comment:
it probably makes sense to also scale this for the CNN models?

Reply:
probably, but let's have this in a separate change.